### PR TITLE
adds a teehee emote

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds a teehee emote

## Why It's Good For The Game

teehee

## Changelog
:cl:
add: teehee
/:cl:

### CODE REVIEW

**Comment by Ryll-Ryll** [comment]
Finally I can RP as Charlotte from making fiends

**Comment by Cyberboss** [comment]
No

**Comment by Cyberboss** [comment]
Fucking stupid 24h policy

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -108,6 +108,15 @@
...
 	message = "winks."
 
+/datum/emote/living/carbon/teehee
+	key = "teehee"
+	key_third_person = "teehees"
+	message = "teehees!"
+	message_mime = "appears to teehee!"
+	emote_type = EMOTE_AUDIBLE
+	sound = 'sound/misc/teehee.ogg'
+	cooldown = TRUE SECONDS
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/emote.dm:118]: amazing

**Comment by ExcessiveUseOfCobblestone** [comment]
Bad-faith PRs do not need to be held to the good-faith rulings we've placed on ourselves.

**Comment by trerri** [comment]
teehee

**Comment by Deafult-Profile** [comment]
teehee

**Pull request closed without merging**